<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTubeStep4</class>
 <widget class="QWidget" name="AddTubeStep4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddTubeStep4</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>样本管下边缘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Picture">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>71</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Save">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_minus_little">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add_more">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_minus_more">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add_little">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
